README.rst

```diff
@@ -1108,6 +1108,22 @@ users (those whose User.type is 'rodsadmin' or 'rodsuser'). The empty group ('em
 members, so it doesn't show up in our final list.
 
 
+iRODS Permissions (ACLs)
+------------------------
+
+The :code:`iRODSAccess` class offers a convenient dictionary interface mapping iRODS permission
+strings to the corresponding integer codes:
+
+>>> from irods.access import iRODSAccess
+>>> iRODSAccess.keys()
+['null', 'read_metadata', 'read_object', 'create_metadata', 'modify_metadata', 'delete_metadata', 'create_object', 'modify_object', 'delete_object', 'own']
+>>> WRITE = iRODSAccess.to_int('modify_object')
+
+Armed with that, we can then query on all data objects with ACL's that allow our user to write them:
+
+>>> from irods.models import DataAccess, DataObject, User
+>>> data_objects_writable =  list(session.query(DataObject,DataAccess,User).filter(User.name=='alice',  DataAccess.type >= WRITE))
```

Comment:
consider...
- spaces after commas
- spaces around `==`
- keep `DataObject, User, DataAccess` in a consistent, teaching-friendly order (code would work, but this is docs)